packages/core/src/utilities/color/color.example.html

```diff
@@ -0,0 +1,28 @@
+<div class="example--list">
```

Comment:
I played around with this example and tried to make the titles more descriptive of the use case for each background color. It involved reworking the templating here quite a bit as well. What do you think?

https://gyazo.com/e5af106e364076c59f7dd4685d7d72a5

Reply:
In case you are interested in using that ^^
```
<section class="example--list">
  <article>
    <h2 class="preview__label">Text on default background (<code>.ds-u-fill--background</code>)</h2>
    <% ['base', 'black', 'gray', 'gray-dark', 'primary', 'primary-darker', 'primary-darkest',
    'primary-alt-darkest', 'error', 'error-dark', 'error-darkest', 'success'].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--background ds-u-color--<%= color %>">.ds-u-color--<%= color %></code>
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on inverse background (<code>.ds-u-fill--background-inverse</code>)
    </h2>
    <% ['white','gray-lightest', 'gray-lighter', 'gray-light', 'primary-alt', 'primary-alt-light',
    'primary-alt-lightest', 'primary-alt-dark'].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--background-inverse ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on lightest gray background (<code>.ds-u-fill--gray-lightest</code>)
    </h2>
    <% ['gray', 'gray-dark', 'base', 'primary', 'primary-darker',
    'primary-darkest','primary-alt-darkest', 'error-dark', 'error-darkest'].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--gray-lightest ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on alert background (<code>.ds-u-fill--primary-alt-lightest</code>)
    </h2>
    <% ['base', 'gray', 'gray-dark', 'primary-darker', 'primary-darkest', 'primary-alt-darkest',
    'error-dark', 'error-darkest' ].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--primary-alt-lightest ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on success alert background (<code>.ds-u-fill--success-lightest</code>)
    </h2>
    <% ['base', 'gray', 'gray-dark', 'primary', 'primary-darker', 'primary-darkest',
    'primary-alt-darkest', 'error-dark', 'error-darkest' ].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--success-lightest ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on warn alert background (<code>.ds-u-fill--warn-lightest</code>)
    </h2>
    <% ['base', 'gray', 'gray-dark', 'primary', 'primary-darker', 'primary-darkest',
    'primary-alt-darkest', 'error-dark', 'error-darkest' ].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--warn-lightest ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
  <article>
    <h2 class="preview__label">
      Text on error alert background (<code>.ds-u-fill--error-lightest</code>)
    </h2>
    <% ['base', 'gray', 'gray-dark', 'primary-darker', 'primary-darkest', 'primary-alt-darkest',
    'error-darkest' ].forEach(color => { -%>
    <article class="ds-u-margin-bottom--1">
      <div
        class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
      ></div>
      <code class="ds-u-fill--error-lightest ds-u-color--<%= color %>"
        >.ds-u-color--<%= color %></code
      >
      <code class="js-swatch-hex c-swatch__label ds-u-color--gray ds-u-fill--transparent">#</code>
    </article>
    <% }) -%>
  </article>
</section>
```